libs/localization/src/lib/en.json

```diff
@@ -1828,6 +1830,7 @@
       "regressionDistributionPivotItem": "Target distribution",
       "metricsVisualizationsPivotItem": "Metrics visualizations",
       "confusionMatrixPivotItem": "Confusion matrix",
+      "AUCPivotItem": "AUC Chart",
```

Comment:
This is fine, but we'll probably need a little explainer somewhere. Especially because it's an acronym

Reply:
will discuss with Minsoo!